databases/week1/Week1-Queries.sql

```diff
@@ -0,0 +1,57 @@
+--1.Find out how many tasks are in the task table
+SELECT count(id) AS task_count
+FROM  task;
+--2.Find out how many tasks in the task table do not have a valid due date
+SELECT count(id) AS task_count
+FROM task
+WHERE due_date IS NULL;
+--3.Find all the tasks that are marked as done
+SELECT * 
```

Comment:
General advice: here instead of selecting all columns and getting a very wide table as a result, i would just select the task title and the status, as it's enough to see the status of each task :)